.cargo/config.toml

```diff
@@ -0,0 +1,44 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+[target.'cfg(all())']
```

Comment:
Is this the same list from main kani

Reply:
32f61e0bd2f757c25c8740370da15a162e949b2d: Turns out only 1 lint-allow is redundant